src/core_pdf/impl/engine/spec/s_08_graphics/icc_profiles.py

```
@@ -266,15 +355,72 @@ def apply_lut_transform(profile: IccLutProfile, samples: ColorSamples) -> ColorS
+    Reduces "unique rows" to a single sort over a flat integer array by packing
+    the channels into one key. Callers must hold the batch to at most four
+    channels, which is what fits in the 32-bit key.
+    """
+    channels = samples.shape[1]
+    if channels > 4:
+        raise IccSampleError("cannot pack more than four channels into one key")
+    keys = numpy.zeros(len(samples), dtype=numpy.uint32)
+    for index in range(channels):
+        keys <<= numpy.uint32(8)
+        keys |= samples[:, index]
+    ignored_keys, first, inverse = numpy.unique(keys, return_index=True, return_inverse=True)
```

Ruff (0.16.2)

[warning] Unpacked variable `ignored_keys` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)